.travis.yml

```diff
@@ -1,24 +1,18 @@
-sudo: required
-dist: trusty
-# Force travis to use its minimal image with default Python settings
+sudo: required 
+dist: trusty 
 language: generic
-compiler:
-  - gcc
-
+notifications:
```

Comment:
> For the sake of consistency!

Constistency does not matter in this case, it is just a preference of the maintainer.

However, I just found it that 
```
  email:
    on_success: change
    on_failure: always
```
is the default (https://docs.travis-ci.com/user/notifications/).
So perhaps it would make sense to set `on_failure: change`.